pallets/pallet-configuration/src/lib.rs

```diff
@@ -0,0 +1,111 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2023 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+//! # Configuration Pallet
+//!
+//! This pallet allows to change configurations without performing a runtime
+//! upgrade.
+//!
+//! Currently the following configurations are supported:
+//!
+//! * `CheckAssociatedRelayNumber` of the parachain-system pallet
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub mod configuration;
+pub mod default_weights;
+
+#[cfg(any(feature = "mock", test))]
+pub mod mock;
+
+#[cfg(feature = "runtime-benchmarks")]
+pub mod benchmarking;
+
+#[cfg(test)]
+/// Tests for the configuration pallet
+mod tests;
+
+pub use crate::{configuration::Configuration, default_weights::WeightInfo, pallet::*};
+
+#[frame_support::pallet]
+pub mod pallet {
+	use super::*;
+
+	use cumulus_pallet_parachain_system::{CheckAssociatedRelayNumber, RelayNumberStrictlyIncreases};
+	use frame_support::{
+		pallet_prelude::*,
+		traits::{EnsureOriginWithArg, StorageVersion},
+	};
+	use frame_system::pallet_prelude::*;
+
+	/// The current storage version.
+	const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
```

Comment:
This should be 0, which is also the default and does not need to be written to storage for new pallets.

Reply:
[27f3605](https://github.com/KILTprotocol/kilt-node/pull/522/commits/27f3605c493f33654099475867b915a388080ed2)